main.yml

```diff
@@ -96,7 +97,7 @@ dashboard:
       - /rhel/dashboard
 
 docs:
```

Comment:
Since `docs` is really vague too, I would love to update this to something like `cs-docs`. I'm open to other naming suggestions too. Any ideas?

Reply:
Good point!